slurm/transfer_targets_fork_main.slurm

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#SBATCH --job-name=tar_transfer         # human-readable label for squeue output
+#SBATCH --time=1:00:00                  # maximum time for this job
+#SBATCH --output=tar_transfer_%u.out    # user's output file
+#SBATCH --partition=cpu                 # which SLURM partition to use
+#SBATCH --nodes=1                       # only request one node
+#SBATCH --ntasks=1                      # only request one core
+#SBATCH --mail-type=ALL                 # enable email notifications for job status. Sends to submitting user
+#SBATCH -A fhwa                         # project account to charge
+
+################################################################################
+#This script syncs the _targets folder and output fles from a forked copy of the repo to the
+#USGS-R/main copy of the repo located at /caldera/projects/usgs/water/impd/fhwa/regional-hydrologic-forcings-ml/
+
+#The directory of the forked repo should be supplied in the sbatch command
+#Example:
+# sbatch --mail-user=<user>@usgs.gov transfer_targets_fork_main.slurm /path/to/user/fork/of/regional-hydrologic-forcings-ml/
+
+#The intended use is to get targets and file data up to date after a PR is merged.
+#Remember to also update the USGS-R/main code with git pull
+################################################################################
+
+module purge
+
+USGS_DIR='/caldera/projects/usgs/water/impd/fhwa/regional-hydrologic-forcings-ml/'
+
+#Checks that the destination target objects are not newer than the newest objects in the source
+#Exits and does not transfer files if they are newer.
+cd $USGS_DIR/'_targets/objects'
+NEWEST_USGS_OBJECT=$(ls -t | head -n1)
+cd $1/'_targets/objects'
+NEWEST_FORK_OBJECT=$(ls -t | head -n1)
+if [ $USGS_DIR'_targets/objects'/$NEWEST_USGS_OBJECT -nt $1/'_targets/objects'/$NEWEST_FORK_OBJECT ]
+then
+    printf '%s\n' "$USGS_DIR has files that are newer than $1 so files are not being copied."
+    exit 1
+fi
```

Comment:
Thanks! I edited the Code / Data Management Plan to add the `tar_outdated` and `tar_make` steps. 

I think for the case when `<forked repo> has files that are newer than $USGS_DIR`, we may not want to transfer files from the fork to USGS main. For example, if the data are newer on the fork as a result of being built from newer code, then merging the data to USGS main would cause a mismatch of the code and data on USGS main. I'm not sure what the best step here would be, but I also do not think this will be common